test/imgcomp.jl

```diff
@@ -15,6 +19,8 @@ using VisualRegressionTests
 
 using Plots
 # using StatPlots
+using PlotReferenceImages
```

Comment:
Maybe just `import PlotReferenceImages`?  You're just using it for `pathof`, right?